### PR TITLE
Change Log

### PR DESCRIPTION
**Pull request by AbrarJahin**
- Gitignore added
- Readme updated
- package version updated

### CODE REVIEW

**Comment by AbrarJahin** [comment]
I signed it!

Please verify my pull request

**Comment by AbrarJahin** [comment]
So, please have a look on this.


Thanks

**Comment by nitobuendia** [comment]
Hello @AbrarJahin,

Thank you so much for the work and big apologies for the huge delay.
I am going to check your changes in the next 72h, but a priori it is looking good.

In the meantime, we have submitted #43, #44 and #45 which updates the node-static version to 0.7.10. Could you update your code to reflect it? We want to leave ^ as you proposed. Hence: ^0.7.10.

Thank you so much.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Logs
```
Comment (nitobuendia) [inline .gitignore]: A lot of these changes do not seem needed. I was able to achieve similar results with a smaller file:
`# Logs
logs
*.log
npm-debug.log*

# Runtime data
pids
*.pid
*.seed
*.pid.lock

# Compiled binary addons (http://nodejs.org/api/addons.html)
build/Release

# Dependency directories
node_modules
*/node_modules
jspm_packages
package-lock.json

# Optional npm cache directory
.npm

# Output of 'npm pack'
*.tgz
`

**Comment by AbrarJahin** [comment]
Hi @nitobuendia , I have solved conflicts and made a commit to merge.

Please check if that is OK.

Thanks for your great help :)

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directories
+node_modules
+*/node_modules
+jspm_packages
+
```
Comment (nitobuendia) [inline .gitignore]: Potentially add .DS_Store

File: step-04/package.json [hunk 3]
```diff
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "description": "WebRTC codelab",
   "dependencies": {
-    "node-static": "0.7.10",
-    "socket.io": "1.2.0"
-  }}
+    "node-static": "^0.7.7",
```
Comment (nitobuendia) [inline step-04/package.json]: ^0.7.10

File: step-05/package.json [hunk 4]
```diff
@@ -3,6 +3,8 @@
   "version": "0.0.1",
   "description": "WebRTC codelab",
   "dependencies": {
-    "node-static": "0.7.10",
-    "socket.io": "1.2.0"
-  }}
+    "node-static": "^0.7.7",
```
Comment (nitobuendia) [inline step-05/package.json]: ^0.7.10

File: step-06/package.json [hunk 5]
```diff
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "description": "WebRTC codelab",
   "dependencies": {
-    "node-static": "0.7.10",
-    "socket.io": "1.2.0"
-  }}
+    "node-static": "^0.7.7",
```
Comment (nitobuendia) [inline step-06/package.json]: ^0.7.10

File: .gitignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (nitobuendia) [reply .gitignore]: Is there any reason I am missing to have all the others?
I do not have anything against them per se.

**Comment by AbrarJahin** [comment]
Hi, @nitobuendia I am creating a new pull request for fixing all of them and closing this one now.
Because I am not finding any way to add my new commit in here

**Comment by AbrarJahin** [comment]
Hi, my new commit is showing in here, so I have fixed everything as you have asked @nitobuendia , please let me know if any further modification needed.

Thanks and happy new year 2018

**Comment by nitobuendia** [comment]
@AbrarJahin Happy New Year! Thanks a lot for the changes.
This is looking good from my side. Sent to Sam for final approval :)

**Comment by samdutton** [comment]
Thanks @AbrarJahin an apologies once again for the delay in responding!

**Pull request merged**